examples/dummy.yaml

```diff
@@ -28,9 +28,6 @@ test_suite_list:
 - suite_dir: test_suite_1
   test_filter_pattern: '*.py'
   test_suite_id: 1
-- suite_dir: test_suite_2
```

Comment:
Why was it removed?

Reply:
Oups, my bad. This was never meant to be push